paddle/fluid/eager/backward.cc

```diff
@@ -643,7 +643,7 @@ std::vector<paddle::experimental::Tensor> RunBackward(
     VLOG(6) << "Running GradNode:" << node->name();
 
     paddle::platform::RecordEvent node_record_event(
-        std::string(typeid(*node).name()) + " grad_node",
```

Comment:
Some performance analysis script rely on this " grad_node" suffix to identify GradNode runtime. Please keep this suffix unchanged.

Reply:
Done